spacemesh/v1/smesher.proto

```diff
@@ -68,30 +60,6 @@ service SmesherService {
         };
     }
 
-    // Get the current min gas for including txs in blocks by this smesher
-    rpc MinGas (google.protobuf.Empty) returns (MinGasResponse) {
-        option (google.api.http) = {
-            post: "/v1/smesher/mingas"
-        };
-    }
-
-    // Set a min gas units for including txs in blocks by this smesher
-    // Returns success if request succeeds, failure if it fails
-    rpc SetMinGas (SetMinGasRequest) returns (SetMinGasResponse) {
-        option (google.api.http) = {
-            post: "/v1/smesher/setmingas"
-            body: "*"
-        };
-    }
-
-    // Estimate smeshing rewards over the next upcoming epoch
-    rpc EstimatedRewards (EstimatedRewardsRequest) returns (EstimatedRewardsResponse) {
```

Comment:
Please also delete the related req/resp types from smesher_types.go etc.

Reply:
Removed unused types and made linting rules stricter, which also lead me to more stuff that could safely be removed.